src/styles/LoginPage.module.scss

```diff
@@ -1,81 +1,69 @@
 @import "colors";
+@import "_variables";
+@import "breakpoints";
+@import "mixins";
 
 @mixin btnStyle {
-  padding: 0.5rem 2rem;
+  padding: $padding-sm-lg;
   background: $white-color;
-  border: 1px solid $black-color;
-  border-radius: 0.25rem;
+  border: $thin-border;
+  border-radius: $border-radius-xs;
   color: $gray-dark--1;
   font-weight: 600;
   cursor: pointer;
   &:hover {
     background-color: $gray-light--3;
-    box-shadow: 2px 4px 4px $gray-light--1;
+    box-shadow: $box-shadow-lg;
   }
 }
 
 .login-page {
   &__container {
-    height: 21.875rem;
+    height: 23.75rem;
```

Comment:
what is the reason to fix the height of the container. can't the content dictate the height of the container? 
I would refrain for using hard values for height, because you would have to change values of every breakpoint then